.github/workflows/release.yml

```diff
@@ -49,17 +47,15 @@ jobs:
           if ${{ github.event.inputs.dry_run }}; then
             echo "options=release:prepare -DdryRun=true -Prelease" >> $GITHUB_ENV
           else
-            echo "options=release:clean release:prepare release:perform -Prelease" >> $GITHUB_ENV
+            echo "options=release:clean release:prepare release:perform" >> $GITHUB_ENV
           fi
 
       - name: Run Maven command
-        # This requires the connection and developerConnection elements in the scm section of the pom
-        # to be set to "scm:git:https:...." thus preventing the release plugin from using SSH.
         run: |
           mvn -B ${{ env.options }}
         env:
-          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
-          MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
+          MAVEN_CENTRAL_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          MAVEN_CENTRAL_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
```

Comment:
I kept the reference to `OSSRH_USERNAME` and `OSSRH_TOKEN` secrets for now, but it would make sense to assign different names to the secrets going forward to avoid confusion.